MyCompany.Seeq.Link.Connector.MyConnector/MyConnection.cs

```diff
@@ -139,6 +139,8 @@ public class MyConnection : ISignalPullDatasourceConnection {
         }
 
         public void Index(SyncMode syncMode) {
+            var rootAssetId = this.createRootAsset();
```

Comment:
I think we should add a comment that we usualy start with creating the root asset as this is needed to create the next level of assets